tests/test_mixins.py

```diff
@@ -1528,6 +1543,59 @@ def test_read_from_postgres_with_query_and_options(self, mock__read_sql):
         # Then
         mock__read_sql.assert_called_with(sql="SELECT * FROM example", con=ANY, parse_dates=["date"])
 
+    @pytest.mark.unit
+    def test_generate_model_from_schema_returns_model(self):
+        # Given
+        postgres_cloud_config = IOConfig(
+            path_to_source_yaml=(os.path.join(constants.TEST_RESOURCES, "definitions/input.yaml")),
+            env_identifier="CLOUD",
+            dynamic_vars=constants,
+        ).get(source_key="READ_FROM_POSTGRES")
+
+        # When
+        schema = postgres_cloud_config["schema"]
+        schema_name = postgres_cloud_config["name"]
+        model = ReadPostgresIO(source_config=postgres_cloud_config)._generate_model_from_schema(schema, schema_name)
+
+        # Then
+        assert len(model.__table__.columns) == len(schema)
+        assert model.__tablename__ == schema_name
+
+    @pytest.mark.unit
+    def test_get_table_columns_from_generated_model_returns_valid_list_of_columns(self, model_expected_columns):
+        # Given
+        pg_cloud_config = IOConfig(
+            path_to_source_yaml=(os.path.join(constants.TEST_RESOURCES, "definitions/input.yaml")),
+            env_identifier="CLOUD",
+            dynamic_vars=constants,
+        ).get(source_key="READ_FROM_POSTGRES")
+
+        # When
+        schema = pg_cloud_config["schema"]
+        schema_name = pg_cloud_config["name"]
+        model = ReadPostgresIO(source_config=pg_cloud_config)._generate_model_from_schema(schema, schema_name)  # pylint: disable=protected-access
+
+        columns = ReadPostgresIO(source_config=pg_cloud_config)._get_table_columns(model)  # pylint: disable=protected-access
+
+        # Then
+        assert isinstance(model.__table__.columns, ImmutableColumnCollection)
+        for x, y in zip(columns, model_expected_columns):
+            assert str(x) == str(y)
```

Comment:
Ok, this one is on me and I am trying to fix it where I can. 

I overuse fixtures at the expense of a test's readability. Instead, I should be creating fixtures only when instantiating an input takes time and it is repeated in many tests. Here, I think this is not the case. 

Again, it is my fault for promoting this bad practice, but kindly, can we avoid using fixtures here and instead replace `model_expected_columns ` with `[PgModel.id, PgModel.foo, PgModel.bar, PgModel.baz]` directly?  I think it improves readability.